src/krate.rs

```diff
@@ -547,13 +549,28 @@ impl Crate {
         }
     }
 
+    // Return None if the documentation URL host matches a blacklisted host
     fn remove_blacklisted_documentation_urls(url: Option<String>) -> Option<String> {
-        let blacklisted_url = "rust-ci.org";
+        let url = match url {
+            Some(url) => url,
+            None => return None,
+        };
```

Comment:
This is such a perfect case for when `?` is supported on `Option`! (nothing to change here, I'm just noting that I wish there was a prettier way to do this but there isn't so this is fine :))